.github/workflows/test.yml

```diff
@@ -28,9 +28,9 @@ jobs:
       - name: rand_distr
         run: cargo doc --all-features --package rand_distr --no-deps
       - name: rand_chacha
-        run: cargo doc --all-features --package rand_chacha --no-deps
+        run: cargo doc --all-features --package rand_chacha --no-deps --features getrandom
```

Comment:
You don't need to specify `--features getrandom` since we already use `--all-features`.